bin/proving-service/README.md

```diff
@@ -14,17 +14,19 @@ To build the service from a local version, from the root of the workspace you ca
 make install-proving-service
 ```
 
-The CLI can be installed from the source code using specific git revisions with `cargo install`. Note that since these aren't official releases we cannot provide much support for any issues you run into, so consider this for advanced users only.
+The CLI can be installed from the source code using specific git revisions with `cargo install` or from crates.io with `cargo install miden-proving-service`.
 
 ## Worker
 
 To start the worker service you will need to run:
 
 ```bash
-miden-proving-service start-worker --host 0.0.0.0 --port 8082
+miden-proving-service start-worker --host 0.0.0.0 --port 8082 --tx-prover --batch-prover
 ```
 
-This will spawn a worker using the hosts and ports defined in the command options. In case that one of the values is not present, it will default to `0.0.0.0` for the host and `50051` for the port.
+This will spawn a worker using the hosts and ports defined in the command options. In case that one of the values is not present, it will default to `0.0.0.0` for the host and `50051` for the port. This command will start a worker that can handle transaction and batch proving requests.
+
+Note that the worker service can be started with only one of the two flags, `--tx-prover` or `--batch-prover`, to handle only transaction or batch proving requests, respectively.
```

Comment:
Why do we impose this limitation? i.e., why not allow the worker to handle heterogeneous requests?

Reply:
Sorry, seems like i miss this doc update from the previous version that I was working on, that use separate workers. This is not a limitation on the current approach. I'm updating this doc.